android/res/values/styles.xml

```diff
@@ -370,12 +370,15 @@
 
     <style name="GetVPNButton">
         <item name="android:background">@drawable/vpn_button_background_selector</item>
-        <item name="android:textColor">#3c4310</item>
+        <item name="android:textColor">@color/white</item>
```

Comment:
Thanks. Yes it is, autocomplete. Will change for the basic_white from FW colors.